slave-services/src/main/java/org/iqu/slaveservices/entities/Event.java

```diff
@@ -4,6 +4,12 @@
 import javax.xml.bind.annotation.XmlAccessorType;
```

Comment:
Shouldn't this class be in the package: **org.iqu.slaveservices.rest.consumer** ,  since it's an rest entity for the slave crawler endpoints.